src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -433,11 +434,16 @@ static ucs_status_t uct_rdamcm_cm_ep_client_init(uct_rdmacm_cm_ep_t *cep,
     ucs_trace("%s: rdma_resolve_addr on cm_id %p",
               uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN),
               cep->id);
-    if (rdma_resolve_addr(cep->id, rdmacm_cm->config.src_addr,
+    if (params->field_mask & UCT_EP_PARAM_FIELD_LOCAL_ADDR) {
+        src_addr = (struct sockaddr*)params->local_addr->addr;
```

Comment:
(const struct sockaddr*)

Reply:
changing to const would cause build error as rdma_resolve_addr need non const